mlflow/models/__init__.py

```diff
@@ -33,3 +36,20 @@ def load(cls, path):
         """Load a Servable from its YAML representation."""
         with open(path) as f:
             return cls(**yaml.safe_load(f.read()))
+
+    @classmethod
+    def log(cls, artifact_path, flavor, **kwargs):
+        """
+        Log model using supplied flavor.
+
+        :param artifact_path: RUN-relative path identifying this model.
+        :param flavor: Flavor that can save the model.
```

Comment:
Can you document the type of `flavor`? It's not obvious.

Reply:
Ok, fair point, I'll make the CondaEnvironment private (I still need something to print out environment with pyspark and python version).